portal/messages/en.json

```
@@ -234,9 +234,16 @@
     "amount": "Amount",
     "drawer": {
       "approving": "Approving {symbol}",
-      "heading": "Review Stake",
-      "subheading": "Review and confirm your stake.",
-      "stake-token": "Stake {symbol}"
+      "stake": {
+        "heading": "Review Stake",
+        "subheading": "Review and confirm your stake."
+      },
+      "stake-token": "Stake {symbol}",
+      "unstake": {
+        "heading": "We’re unlocking your tokens",
+        "subheading": "The network is processing your unlock. You’ll be able to access your tokens once the step is complete."
```

Using Step sounds weird 🤔 

what do you think about


```suggestion
        "subheading": "The network is processing your unlock. You’ll be able to access your tokens once the transaction is complete."
```

?